packages/ui-components/src/lib/providers/filters/FilterStoreProvider.svelte

```diff
@@ -13,9 +13,19 @@
 		chainIds: undefined
 	};
 
+	export const DEFAULT_ORDER_FILTERS: GetOrdersFilters = {
+		owners: [],
+		active: undefined,
+		orderHash: undefined,
+		tokens: undefined,
+		chainIds: undefined
+	};
+
 	export type FilterStoreContext = {
 		filterStore: Readable<RaindexFilterStore>;
 		currentVaultsFilters: Readable<GetVaultsFilters>;
+		currentOrdersFilters: Readable<GetOrdersFilters>;
+		isLoaded: Readable<boolean>;
 	};
```

Comment:
@findolor you have asked me before why do we need Svelte stores here.
So that the answer doesn't get lost in chat, I'll answer here.

When any of filters are updated we want to re-render GUI.
Therefore GUI components should listen for changes in the FilterStore.
To have less coupling and cleaner code I've created "reactive selectors" instead of having subscription on the whole FilterStore and later calling getters. It follows Svelte reactivity models and therefore I've decided to use Svelte store.

Any objections?